src/ExtensionsSample/Program.cs

```diff
@@ -54,19 +55,14 @@ public static void Main(string[] args)
             // be indexed by the JobHost.
             // To run some of the other samples included, add their types to this list
             config.TypeLocator = new SamplesTypeLocator(
-                typeof(ErrorMonitoringSamples),
-                typeof(FileSamples),
-                typeof(MiscellaneousSamples),
-                typeof(SampleSamples),
-                typeof(TableSamples),
-                typeof(TimerSamples));
+                typeof(DocumentDBSamples));
 
             // Some direct invocations to demonstrate various binding scenarios
-            host.Call(typeof(MiscellaneousSamples).GetMethod("ExecutionContext"));
-            host.Call(typeof(FileSamples).GetMethod("ReadWrite"));
-            host.Call(typeof(SampleSamples).GetMethod("Sample_BindToStream"));
-            host.Call(typeof(SampleSamples).GetMethod("Sample_BindToString"));
-            host.Call(typeof(TableSamples).GetMethod("CustomBinding"));
+            //host.Call(typeof(MiscellaneousSamples).GetMethod("ExecutionContext"));
+            //host.Call(typeof(FileSamples).GetMethod("ReadWrite"));
+            //host.Call(typeof(SampleSamples).GetMethod("Sample_BindToStream"));
+            //host.Call(typeof(SampleSamples).GetMethod("Sample_BindToString"));
+            //host.Call(typeof(TableSamples).GetMethod("CustomBinding"));
```

Comment:
why are these commented out?

Reply:
Oh that's just for my testing -- I usually change the loaded sample to only use the class that I'm testing and so these always fail while I'm I'm playing around. I usually back that out in the last second, but must have missed it this time.